src/Symfony/Bridge/Twig/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": ">=5.3.9",
         "symfony/security-csrf": "~2.6|~3.0.0",
-        "twig/twig": "~1.13,>=1.13.1"
+        "twig/twig": "~1.17"
```

Comment:
and the root composer.json also needs to be updated with the new Twig constraint